nixos/modules/services/networking/url-bot-rs.nix

```diff
@@ -0,0 +1,131 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.url-bot-rs;
+
+  format = pkgs.formats.toml {};
+  configFile = format.generate "url-bot-rs.toml" cfg.settings;
+in
+{
+  options.services.url-bot-rs = {
+    enable = mkEnableOption "Minimal IRC URL bot in Rust";
+
+    extraArgs = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      example = [ "--verbose" ];
+      description = ''
+        List of additional command line parameters for url-bot-rs.
+      '';
+    };
+
+    settings = mkOption {
+      type = format.type;
+      default = {};
+      description = ''
+        Configuration for url-bot-rs, see https://github.com/nuxeh/url-bot-rs#configuration-file-options
```

Comment:
Users will love you if you make this link clickable :heart:

Reply:
Thanks! Done in [`1378085` (#105315)](https://github.com/NixOS/nixpkgs/pull/105315/commits/137808582d79c2f5a2d6cfb8417da9fc76ac5869).